### PR TITLE
Quickstart Tweaks to use deb/rpm repo

### PR DESCRIPTION
**Pull request by benarent**
This PR updates install and quick start with rpm/deb instructions from https://rpm.releases.teleport.dev/ & https://deb.releases.teleport.dev/  to improve content.

### CODE REVIEW

File: docs/5.0/pages/installation.md [hunk 1]
```diff
@@ -14,57 +14,24 @@ The following examples install the 64-bit version of Teleport binaries, but
...
+    # Add repo to APT
+    $ add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+    # Update APT Cache
+    $ apt update
+    # Install Teleport
+    $ apt install teleport
     ```
 
-=== "DEB"
+=== "rpm repo & Amazon Linux 2"
```
Comment (webvictim) [inline docs/5.0/pages/installation.md:30]:
```suggestion
=== "RPM repo (CentOS/RHEL/Fedora/Amazon Linux 2)"
```

File: docs/5.0/pages/installation.md [hunk 2]
```diff
@@ -14,57 +14,24 @@ The following examples install the 64-bit version of Teleport binaries, but
 Release](https://gravitational.com/teleport/download/) page for the most
 up-to-date information.
 
-=== "Tarball"
+=== "deb repo"
```
Comment (webvictim) [inline docs/5.0/pages/installation.md:17]:
```suggestion
=== "DEB repo (Debian/Ubuntu/Linux Mint)"
```

File: docs/5.0/pages/quickstart.md [hunk 3]
```diff
@@ -19,17 +19,20 @@ Teleport on Linux machine(s).
 There are several ways to install Teleport.
 Take a look at the [Teleport Installation](installation.md) page to pick the most convenient for you.
 
-=== "yum repo / AWS Linux 2"
+=== "yum repo"
```
Comment (webvictim) [inline docs/5.0/pages/quickstart.md:22]:
```suggestion
=== "RPM repo (CentOS/RHEL/Fedora/Amazon Linux 2)"
```

File: docs/5.0/pages/quickstart.md [hunk 4]
```diff
@@ -19,17 +19,20 @@ Teleport on Linux machine(s).
...
     ```bash
     sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
     sudo yum install teleport
-
-    # Optional:  Using DNF on newer distributions
-    # sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-    # sudo dnf install teleport
     ```
 
+=== "deb repo"
```
Comment (webvictim) [inline docs/5.0/pages/quickstart.md:29]:
```suggestion
=== "DEB repo (Debian/Ubuntu/Linux Mint)"
```

File: docs/5.0/pages/quickstart.md [hunk 5]
```diff
@@ -19,17 +19,20 @@ Teleport on Linux machine(s).
...
-=== "yum repo / AWS Linux 2"
+=== "yum repo"
 
     ```bash
     sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
     sudo yum install teleport
-
-    # Optional:  Using DNF on newer distributions
-    # sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-    # sudo dnf install teleport
```
Comment (webvictim) [inline docs/5.0/pages/quickstart.md:30]: They've actually removed yum completely on newer versions of Fedora, so if if isn't too much of a pain I'd suggest leaving this in.

**Review by webvictim — APPROVED** [review]

File: docs/5.0/pages/quickstart.md (hunk 5 above)
Comment (benarent) [reply docs/5.0/pages/quickstart.md]: Good call, I've added it back.

File: docs/5.0/pages/installation.md [hunk 6]
```diff
@@ -14,57 +14,24 @@ The following examples install the 64-bit version of Teleport binaries, but
 Release](https://gravitational.com/teleport/download/) page for the most
 up-to-date information.
 
-=== "Tarball"
+=== "DEB repo (Debian/Ubuntu/Linux Mint)"
```
Comment (klizhentas) [inline docs/5.0/pages/installation.md:17]: `DEB (Debian/Ubuntu)`

File: docs/5.0/pages/installation.md [hunk 7]
```diff
@@ -14,57 +14,24 @@ The following examples install the 64-bit version of Teleport binaries, but
...
-    # <checksum> <filename>
-    $ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-    $ shasum -a 256 teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-    # Verify that the checksums match
-    $ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-    $ cd teleport
-    $ ./install
-    $ which teleport
-    /usr/local/bin/teleport
+    # Install our public key. Fingerprint: 0c5e 8ba5 658e 320d 1b03 1179 c87e d53a 6282 c411
```
Comment (klizhentas) [inline docs/5.0/pages/installation.md:20]: Remove fingerprint, we could change it in the future, HTTPS is source of trust here.

File: docs/5.0/pages/installation.md [hunk 8]
```diff
@@ -14,57 +14,24 @@ The following examples install the 64-bit version of Teleport binaries, but
...
+    $ curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
+    # Add repo to APT
+    $ add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+    # Update APT Cache
+    $ apt update
```
Comment (klizhentas) [inline docs/5.0/pages/installation.md:25]: @webvictim  not sure if `apt` is installed by default on Debian and Mint, let's do `apt-get`

File: docs/5.0/pages/installation.md [hunk 9]
```diff
@@ -14,57 +14,24 @@ The following examples install the 64-bit version of Teleport binaries, but
...
+    # Install Teleport
+    $ apt install teleport
     ```
 
-=== "DEB"
+=== "RPM repo (CentOS/RHEL/Fedora/Amazon Linux 2)"
```
Comment (klizhentas) [inline docs/5.0/pages/installation.md:30]: RPM (without repo) (Amazon Linux 2/RHEL) (Fedora and CentOS are actually EOL)

File: docs/5.0/pages/installation.md [hunk 10]
```diff
@@ -93,13 +58,24 @@ up-to-date information.
...
+    # Verify that the checksums match
+    $ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    $ cd teleport
+    $ ./install
     ```
 
 ## Docker
 
-Please follow our [OSS Docker Quickstart](quickstart-docker.md) or [Enterprise Docker Quickstart](enterprise/quickstart-enterprise.md#run-teleport-enterprise-using-docker) for install and setup instructions.
+Please follow our [OSS Docker Quick Start](quickstart-docker.md) or [Enterprise Docker Quick Start](enterprise/quickstart-enterprise.md#run-teleport-enterprise-using-docker) for install and setup instructions.
```
Comment (klizhentas) [inline docs/5.0/pages/installation.md:78]: Let's use `Getting started` as a phrase, see https://goteleport.com/teleport/docs/docs

File: docs/5.0/pages/quickstart.md [hunk 11]
```diff
@@ -19,17 +19,24 @@ Teleport on Linux machine(s).
 There are several ways to install Teleport.
 Take a look at the [Teleport Installation](installation.md) page to pick the most convenient for you.
 
-=== "yum repo / AWS Linux 2"
+=== "RPM repo (CentOS/RHEL/Fedora/Amazon Linux 2)"
```
Comment (klizhentas) [inline docs/5.0/pages/quickstart.md:22]: same

**Review by klizhentas — CHANGES_REQUESTED** [review]

File: docs/5.0/pages/installation.md (hunk 9 above)
Comment (webvictim) [reply docs/5.0/pages/installation.md:30]: You're right that CentOS is EOL but Fedora isn't

File: docs/5.0/pages/installation.md (hunk 6 above)
Comment (benarent) [reply docs/5.0/pages/installation.md:17]: Updated to lead with Debian/Ubuntu.

File: docs/5.0/pages/installation.md (hunk 10 above)
Comment (benarent) [reply docs/5.0/pages/installation.md]: Updated these links and modified the language on the quickstart page.

**Review by klizhentas — APPROVED** [review]

**Pull request merged**